src/graph/GraphViewsManager.ts

```diff
@@ -69,41 +104,180 @@ export class GraphViewsManager implements IServerProvider {
     this._servers.set(id, server);
     return id;
   }
+
+  public findPanelById(id: number): vscode.WebviewPanel {
+    return this._panels.get(id);
+  }
+
 }
 
-class GraphViewDocumentContentProvider implements vscode.TextDocumentContentProvider {
+class GraphViewDocumentContentProvider {
   public onDidChange?: vscode.Event<vscode.Uri>;
 
-  public constructor(private _serverProvider: IServerProvider) { }
+  public constructor(private _serverProvider: IServerProvider, private _context: vscode.ExtensionContext) { }
 
-  public provideTextDocumentContent(uri: vscode.Uri, _token: vscode.CancellationToken): vscode.ProviderResult<string> {
+  public provideHtmlContent(uri: vscode.Uri): string {
+    // the uri has format: "1", "2" ...
     // Figure out which client to attach this to
     // tslint:disable-next-line:no-single-line-block-comment
-    let serverId = parseInt(uri.path.slice(1) /* remove '/' from beginning */, 10);
+    let serverId = parseInt(uri.path, 10);
     console.assert(serverId > 0);
     let server = this._serverProvider.findServerById(serverId);
     if (server) {
-      let clientHtmlPath = path.join(resourcesPath, 'graphClient', 'graphClient.html');
-      console.assert(fs.existsSync(clientHtmlPath), `Couldn't find ${clientHtmlPath}`);
+      let options: IGraphClientHtmlPaths = {
```

Comment:
nit: it be easier just to pass in the base path for these as a single variable?  I don't think web browser care whether the '/' and '\' are mismatched.  Could be wrong, though.

Reply:
That's what I was saying - I don't think Chrome cares.